prompt_toolkit/utils.py

```diff
@@ -261,6 +262,14 @@ def take_using_weights(
         i += 1
 
 
+def to_type(value: Union[Callable[[], _T], _T]) -> _T:
```

Comment:
Can you rename it to something like ``call_if_callable``. I think the naming is a little confusing.

Reply:
Renamed